packages/rest/tests/e2e/member.spec.ts

```diff
@@ -54,7 +54,9 @@ describe('Member tests', () => {
     expect(member2.nick).to.null
   })
 
-  it('Send a direct message', async () => {
+  it('Send a direct message', async function () {
```

Comment:
why change from arrow function to normal function?

Reply:
`this`.

for a more serious explanation: An arrow function does have have a this itself, and mocha gives a this for configure the timeout